devTools/phpstan-tests.neon

```diff
@@ -1,7 +1,11 @@
 parameters:
+    level: 4
```

Comment:
can't it be max?

Reply:
It was already level 4 before hand, so i've decided not to change it for now